### ISSUE TITLE
Changes needed for TAC/TOB data

### ISSUE THREAD

**Issue by joe-pitt** [mrghg/agage-archive#88]
In order to get monthly means calculated with the pollution removed, we need the following things to happen:

- Peter Salameh syncs the TAC MD and the TOB ECD to hubibti
- Ray Wang calculates pollution flags for TAC MD, TOB ECD and TAC LGR

At the moment the TAC LGR is synced, so Peter creates the .nc files with a pollution flag field, but there are no pollution flags calculated. This is why we get the following errors at the moment:

error_log_individual.txt
```
TAC n2o: ValueError in stack: run (line 251) / io (line 1026). No data retained for n2o TAC lgr after applying release schedule end dates. Check dates in release schedule or omit this instrument.
TAC co: ValueError in stack: run (line 251) / io (line 1026). No data retained for co TAC lgr after applying release schedule end dates. Check dates in release schedule or omit this instrument.
```

error_log_combined.txt
```
TAC n2o: ValueError in stack: run (line 437) / io (line 1026). No data retained for n2o TAC  after applying release schedule end dates. Check dates in release schedule or omit this instrument.
TAC co: ValueError in stack: run (line 437) / io (line 1026). No data retained for co TAC  after applying release schedule end dates. Check dates in release schedule or omit this instrument.
```

**Comment by mrghg** [comment]
Do we need the baseline for this site? Perhaps I should change the run script to exclude baselines for selected sites?

**Comment by joe-pitt** [comment]
Hmmm not sure, although we have baselines for TAC/TOB for the other species, so it might make sense to get them for these species too?

**Comment by joe-pitt** [comment]
Ray Wang has added the LGR flags. Let's discuss the TAC MD and TOB ECD at the December AGAGE meeting

**Comment by joe-pitt** [comment]
Will push these files to Peter once he has his new hubibti data transfer regime set up

**Comment by joe-pitt** [comment]
Peter is still working on the hubibti setup. I don't think we're going to get these data synced and pollution flags calculated in time for this release. Maybe we should withhold this data for now and try to get in included in the next release. What do you reckon @mrghg @Amsoht ?

**Comment by Amsoht** [comment]
are we talking about taunus-ecd only? I think it's ok to withhold the monthly mean data for this release.

**Comment by joe-pitt** [comment]
Yeah this is the taunus-ecd sf6. So no monthly means, but do we still want to release the high frequency data (and include it in the combined file) without pollution flags?

**Comment by joe-pitt** [comment]
Note that there are actually no pollution flags for the TAC LGR either - Ray said he is happy to add these but at the moment this instrument is not on his list to release.

**Comment by joe-pitt** [comment]
Agreed via email that we will withhold these from the Jan/Feb 2025 release

**Comment by joe-pitt** [comment]
Decided at the April 2026 European AGAGE data review that we would like to release the TAC LGR and TAC MD. Kieran is following up via email regarding the appropriate process to get these approved for release as AGAGE data
